doc/library/tensor/nnet/conv.txt

```diff
@@ -70,61 +70,27 @@ TODO: Give examples on how to use these things! They are pretty complicated.
 
 - Implemented operators for neural network 2D / image convolution:
     - :func:`nnet.conv.conv2d <theano.tensor.nnet.conv.conv2d>`.
-      CPU convolution implementation, previously used as the convolution interface.
-      This is the standard operator for convolutional neural networks working
-      with batches of multi-channel 2D images, available. It
-      computes a convolution, i.e., it flips the kernel.
+      CPU convolution implementation, previously used as the
+      convolution interface.  This is the standard operator for
+      convolutional neural networks working with batches of
+      multi-channel 2D images, available. It computes a convolution,
+      i.e., it flips the kernel.
       Most of the more efficient GPU implementations listed below can be
       inserted automatically as a replacement for nnet.conv.conv2d via graph
       optimizations. Some of these graph optimizations are enabled by default,
       others can be enabled via Theano flags.
-      Since November 24th, 2014, you can also use a meta-optimizer to
-      automatically choose the fastest implementation for each specific
-      convolution in your graph using the old interface. For each instance,
-      it will compile and benchmark each applicable implementation of the ones
+      You can also use a meta-optimizer to automatically choose the
```

Comment:
I would just update the doc that we have that and we plan to port it instead of removing? No strong feeling what is better, just a suggestion.